libs/designsystem/src/lib/helpers/design-token-helper.ts

```diff
@@ -48,6 +47,16 @@ export class DesignTokenHelper {
     return styles.fontSizes[key];
   }
 
+  public static iconFontSize(key: 'xs' | 'sm' | 'md' | 'lg'): string {
+    const iconSizeMapping = {
+      xs: 's',
+      sm: 'm',
+      md: 'l',
+      lg: 'xxxl',
+    };
```

Comment:
Consider moving `$_icon-font-size-map` from `libs/designsystem/src/lib/components/icon/_icon.component.shared.scss` into `libs/designsystem/src/lib/scss/base/_variables.scss` to get this map auto-generated  to `libs/designsystem/src/lib/helpers/design-token-helper.styles.ts`

Reply:
Fixed in 1ed32aae47aa9cefab2ea94b68df094a5dbcc92b where I also simplified the use of the variables a bit, so we use `map-get` directly instead of a helper method that does essentially the same.